Tests/Expected/Storyboards-iOS/swift2-context-all-ignore-module.swift

```diff
@@ -109,7 +109,7 @@ enum StoryboardSegue {
     case CustomBack
     case Embed
     case NonCustom
-    case Show_NavCtrl = "Show-NavCtrl"
+    case ShowNavCtrl = "Show-NavCtrl"
```

Comment:
Does that mean that the next release is gonna be breaking?
I'm OK with that — the change is good here — but we'll have to wonder if that should mean bumping SwiftGen to major for that too…